helm/alfresco-content-services/values.yaml

```diff
@@ -918,6 +913,8 @@ alfresco-sync-service:
   # Check [Alfresco Sync Service
   # Documentation](https://github.com/Alfresco/alfresco-helm-charts/tree/main/charts/alfresco-sync-service)
   enabled: true
+  image:
+    tag: 4.0.0-M7
```

Comment:
I have the feeling we need a new epic (?) for moving/splitting the current updatecli to the new helm charts repo (one updatecli config for each chart?) and avoid having any image tag in the parent chart (which of course can be overridden here by users)